.gitignore

```diff
@@ -1,3 +1,4 @@
 .vagrant
 vagrant_ansible_inventory_default
 tests.pyc
+secret
```

Comment:
Shouldn't you be using [ansible-vault](http://docs.ansible.com/ansible/playbooks_vault.html) for secrets? Then you can safely commit them to the repo.